calcos/fpavg.py

```diff
@@ -1200,10 +1200,12 @@ def normalizeSums(self, data, sumweight):
         error_frequentist_lower, error_frequentist_upper = cosutil.errFrequentist(variance)
         error_frequentist_lower[zeroweight] = 0.0
         error_frequentist_upper[zeroweight] = 0.0
-        conversion = data.field("flux") / data.field("net")
+        bad = np.where(data.field("net") == 0.0)
+        good = np.where(data.field("net") != 0.0)
+        nelements = len(data.field("flux"))
+        conversion = np.zeros(nelements)
```

Comment:
Is this correct? `np.zeros` takes a shape and here a length is passed. The length is 2 for 2D data because this is the output of `np.where`. So do you need the shape here?

Reply:
These are 1-d arrays, conversion is created as an array of zeros with the same number of elements as the "flux" and "net" arrays so that the "bad" and "good" indices determined from where "net" is or isn't == 0.0 can be applied to the conversion array

Reply:
OK, thanks for the clarification.